### PR TITLE
Read - Completed

### PR DESCRIPTION
**Pull request by Valeriano-WebDev**

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -16,7 +16,24 @@ import {
...
+      profiles: []
+    }
+  }
+
+  componentDidMount() {
+    this.readProfile()
+  }
+
+  readProfile = () => {
+    fetch("http://localhost:3000/profiles")
```
Comment (egomez-papa) [inline app/javascript/components/App.js:31]: create a config file to hold the URL.

Also; you can shorten it to just "/profiles"
And it'll assume the default URL is localhost

**Review by egomez-papa — CHANGES_REQUESTED** [review]
Looking good.

File: app/javascript/components/App.js (hunk 1 above)
Comment (Valeriano-WebDev) [reply app/javascript/components/App.js:31]: Thank you, will shorten the link up now.

Comment (mrcallenmatias) [reply app/javascript/components/App.js]: thanks for the tip!

**Review by egomez-papa — APPROVED** [review]
LGTM 🚀

**Review by scalisema — APPROVED** [review]

**Pull request merged**